near-sdk-macros/src/core_impl/code_generator/impl_item_method_info.rs

```diff
@@ -151,7 +151,7 @@ impl ImplItemMethodInfo {
             #non_bindgen_attrs
             #[cfg(target_arch = "wasm32")]
             #[no_mangle]
-            pub extern "C" fn #ident() {
```

Comment:
I think `extern "C"` should stay, it protects you from accidently exposing the undercommunted and explicitly unstable details of the ABI of the partciular compiler version:

```
14:59:07|~/tmp
λ rustc lib.rs --crate-type cdylib --target wasm32-unknown-unknown -C link-arg=-s
warning: `extern` fn uses type `S`, which is not FFI-safe
 --> lib.rs:4:24
  |
4 | pub extern "C" fn f(_: S) {}
  |                        ^ not FFI-safe
  |
  = note: `#[warn(improper_ctypes_definitions)]` on by default
  = help: consider adding a `#[repr(C)]` or `#[repr(transparent)]` attribute to this struct
  = note: this struct has unspecified layout
note: the type is defined here
 --> lib.rs:1:1
  |
1 | pub struct S(u64, u64);
  | ^^^^^^^^^^^^^^^^^^^^^^^

warning: 1 warning emitted


14:59:09|~/tmp
λ micro lib.rs

14:59:17|~/tmp
λ bat -p lib.rs
pub struct S(u64, u64);

#[no_mangle]
pub fn f(_: S) {}

14:59:23|~/tmp
λ rustc lib.rs --crate-type cdylib --target wasm32-unknown-unknown -C link-arg=-s
```

Reply:
ah, interesting, was not aware. Will add back